### DISCUSSION TITLE
IPX fixes

### DISCUSSION THREAD

**Discussion by jharrison03** [dosemu2/dosemu2#1792] [Ideas]
I see this is still being maintained as part of the networking, but IPX has been gone from the kernel for sometime now.  I use IPX, but I do this client-side from within DOSemu with a bridged network, and anyhow, it's always been faster than actually using the native IPX support from the kernel for whatever reason.

Just my observation.

**Comment by stsp** [comment]
We have at least 1 IPX user @bpranoto
who regularly tests it so it can't be dropped
just yet.
As for being slower - this may be something
to investigate, as the kernel support is
going to be faster.
I would recommend re-checking as
recently there were critical networking
fixes that also affect performance.
Hope IPX is still ok after them, and it
could get faster.
